### ISSUE TITLE
[question] inline svg with role="img" and aria-hidden

### ISSUE THREAD

**Issue by nico3333fr** [ffoodd/a11y.css#387]
While testing ProtonMail v4, I found maybe one issue: 

- it says that a SVG> is missing aria-label or aria-labelledby

For this code:

`<svg viewBox="0 0 16 16" class="icon-24p" role="img" aria-hidden="true" focusable="false">
  <use xlink:href="#shape-twitter"></use>
</svg> Our Twitter account`
(example here: https://design-system.protontech.ch/icons/#integration-cases)

Accede-web says we should use `aria-hidden` https://www.accede-web.com/notices/html-et-css/images-et-icones/gerer-lalternative-des-svg-images-vectorielles/#ancre-01

Who should I believe? :) 
Or maybe restrict the test to elements that are not `aria-hidden` ?

Cheers and thanks for advice ;)

Btw, some links are broken https://ffoodd.github.io/a11y.css/errors.html#no-aria-label

**Comment by ffoodd** [comment]
I think we should exclude `aria-hidden` 👌

**Comment by PhilippeVay** [comment]
Sidenote to PR: links doesn't seem broken anymore
